git-dependencies

```diff
@@ -62,10 +69,13 @@ def create_parser():
 									 description='Manage dependencies in git repositories',
 									 formatter_class=argparse.RawTextHelpFormatter)
 	parser.add_argument('-r', '--recursive', action='store_true', help='recurse into dependencies')
+	parser.add_argument('-o', '--overrides', help='override dependencies from file')
+	parser.add_argument('-oa', '--override-all', action='store_true', help='enforce complete override (fail if a dependency is not overridden)')
```

Comment:
If I understood well, if I want to override dependencies on only 1 level `-overrides` argument is enough, but if I want to override all of them I have to add `--override-all` argument too. Could you please confirm that?
Would it be possible to use `--recursive` argument to reach the same functionality?